AUTHORS

```diff
@@ -0,0 +1,19 @@
+The Prometheus project was started by Matt T. Proud in 2012.
```

Comment:
Oh, and I forgot to mention that Matt explicitly told me he wanted us to add a notice here that he is now an alumni/emeritus, so he doesn't get a lot of mail/contacts about it. Maybe we should just put something like this behind his name: "Matt T. Proud (now emeritus)". Or something like that.